configure.ac

```diff
@@ -202,6 +202,19 @@ AC_CHECK_HEADERS(fcntl.h unistd.h bfd.h)
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
 
+HAVE_BFD_XTENSA_PDIFF_RELOCS=0
+case $target in
+	xtensa*)
+		AS_VAR_COPY([OLD_CFLAGS], [CFLAGS])
```

Comment:
any reason for using CFLAGS instead of CPPFLAGS ?  includes (-I) are CPPFLAGS, not CFLAGS.

Reply:
No reason, you're right. I've changed it to use CPPFLAGS.